docs/topics/serialization.txt

```diff
@@ -268,6 +268,15 @@ work::
                 return force_text(obj)
             return super(LazyEncoder, self).default(obj)
 
+By default, :class:`~django.core.serializers.json.DjangoJSONEncoder` is used
+for JSON encoding/decoding.  It should also be used as a parent class when
+creating your own encoding class, as in the example above.
+:class:`~django.core.serializers.json.DjangoJSONEncoder` will handle
```

Comment:
will handle -> handles
You might convert the comma list to a bulleted list and add a description of how each type is serialized